WordPress/Classes/Models/SharingButton+Lookup.swift

```diff
@@ -0,0 +1,32 @@
+extension SharingButton {
+
+    /// Returns an array of all cached `SharingButtons` objects.
+    ///
+    /// - Parameters
+    ///     - blog: A `Blog` object
```

Comment:
Nitpick: This documentation uses a different format that the one in the file before. Which one should we use?

Regardless, how would you feel about removing it? Unlike the one above, that explains the relationship between name and `serviceID`, this one doesn't explain much.

```suggestion
```

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20265/commits/e95fd668a9f77923ccf029922b0b0b4366e21d74